### ISSUE TITLE
The `-R` option is broken on Ubuntu/Debian systems

### ISSUE THREAD

**Issue by UtahDave** [saltstack/salt-bootstrap#1081]
When attempting to use the `-R` option the bootstrap script fails to use the provided repo and ends up installing from the default distro repos which results in Salt version 0.17.5.

I tested this on the latest bootstrap script downloaded like this:
```
curl -L https://bootstrap.saltstack.com -o install_salt.sh
```
I tested this on Ubuntu 14.04

### Setup
Even using the main `repo.saltstack.com` repo with the option fails

```
./install_salt.sh -D -R repo.saltstack.com -A 10.181.130.1
```
```
root@testbootstrap:~# salt-minion --version
salt-minion 0.17.5
```

**Comment by UtahDave** [comment]
ZD 1473

**Comment by rallytime** [comment]
@UtahDave I have fixed this with PR #1082. Thanks for filing this and letting me know!
```
root@ubuntu:~# sh test.sh -R repo.saltstack.com/staging
<snipped>
root@ubuntu:~# salt-minion --version
salt-minion 2016.11.5 (Carbon)
root@ubuntu:~# cat /etc/apt/sources.list.d/saltstack.list
deb https://repo.saltstack.com/staging/apt/ubuntu/14.04/amd64/latest trusty main
```
